libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.kt

```diff
@@ -168,142 +210,114 @@ open class LoginActivity : AppCompatActivity(), OAuthWebviewHelperEvents {
             onRequestContinued()
         }
 
-        setTheme(
-            when (salesforceSDKManager.isDarkTheme) {
-                true -> SalesforceSDK_Dark_Login
-                else -> SalesforceSDK
-            }
-        )
-
-        salesforceSDKManager.setViewNavigationVisibility(this)
-
-        // Determine login options for Salesforce Identity API UI Bridge front door URL use or choose defaults.
-        val loginOptions = when {
-            isUsingFrontDoorBridge -> salesforceSDKManager.loginOptions
-            else -> fromBundleWithSafeLoginUrl(intent.extras)
-        }
-
         // Protect against screenshots
-        window.setFlags(FLAG_SECURE, FLAG_SECURE)
-
-        /*
-         * Fetch authentication configuration except when using Salesforce Identity API UI Bridge, since the front door URL may be for another login server.
-         *
-         * Support for adding new login servers from the front door bridge URL is not yet implemented.  Unknown login servers will fail login.
-         */
-        if (!isUsingFrontDoorBridge) {
-            salesforceSDKManager.fetchAuthenticationConfiguration()
-        }
-
-        // Setup content view
-        setContentView(sf__login_layout)
-        if (salesforceSDKManager.isIDPLoginFlowEnabled) {
-            findViewById<Button>(sf__idp_login_button).visibility = VISIBLE
+        if (!SalesforceSDKManager.getInstance().isDebugBuild) {
+            window.setFlags(FLAG_SECURE, FLAG_SECURE)
```

Comment:
I see most of our internal apps removing the flag for debug builds and we have gotten this request many time externally.